datastore/snippets/snippet_test.go

```diff
@@ -534,9 +534,10 @@ func SnippetIterator_Cursor() {
 
 	// Get the cursor for the next page of results.
 	nextCursor, err := it.Cursor()
+	// When run within a loop, this yields the cursor to continue reading results
```

Comment:
I don't understand this sample. There is no loop, but there should be? Should we add one?

I'm confused -- `cursorStr` is set to `""`, then immediately checked `if cursorStr != ""`.

Reply:
My thinking for the sample was simply to show all variable initialization, but it could make sense to actually set up a loop. I'll have to further examine the implications for tests, however.

Reply:
I've updated this sample to exist within a loop, but I'm not actually certain this samples repo is set up to test everything like it claims.